modules/pruning-images.adoc

```diff
@@ -25,20 +20,19 @@ When the Pruning Custom Resource is deleted, the pruning `CronJob` and its relat
 [source,yaml]
 ----
 spec:
-  schedule: "0 0 * * *" <1>
+  schedule: 0 0 * * * <1>
   suspend: false <2>
   keepTagRevisions: 3 <3>
-  keepYoungerThan: 60m <4>
+  keepYoungerThan: 3600000000000 <4>
```

Comment:
Should this be 3600000000000s?

Reply:
No, just an integer is fine.

Reply:
It should be an integer (so it cannot have suffixes), and it's in **nano**seconds.

Reply:
Ah, thanks!